ddp-workspace/projects/ddp-dsm-ui/src/app/shipping/shipping.component.ts

```diff
@@ -276,6 +279,9 @@ export class ShippingComponent implements OnInit {
               if (kit.noReturn) {
                 this.kitsWithNoReturn = true;
               }
+              if(kit.receivedBy === 'MERCURY') {
+                this.isClinicalStudy = true;
+              }
```

Comment:
If I understood the means behind this correctly, I think this means that before getting any kits received by "Mercury", the `this.isClinicalStudy` will always be false, which means that we won't be seeing `SampleType` column. Can't we use instance settings for this?

Reply:
Mercury is the main receiver for this study, so as soon as we receive 1 "normal" kit for this study we will display the info. no need to add an additional call to the backend to get the information from instance settings